DFE-Digital/academies-academisation-api
# AB#105435 - conversion target date - store whether target date has be…

https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_workitems/edit/105435?McasTsid=26110

## Type of change

## Expected behaviour
Extension to school object - add bool? WantConversionToHappenOnSelectedDate = null,

## Steps to reproduce issue (if relevant)
n/a

## Steps to test this PR
1. Run Unit tests to ensure tests not broken - done by CI/CD
2. Run dotnet ef database update to ensure new field gets added to the database
3. Run API to ensure GET && PUT include new property

## Prerequisites
n/a
